README.md

```diff
@@ -154,6 +154,19 @@ Training takes around 1.5 days with 8 NVIDIA V100 GPUs, you can download a pretr
 
 After pretraining, a checkpoint is saved in ```exps/DETReg_top30_in100/checkpoint.pth```. To fine tune it over different coco settings use the following commands:
 
+### Pretraining on ImageNet with pseudo labels from MDETR / MDef-DETR proposals
+
+[comment]: <> (The MDETR proposals generated on ImageNet100 is saved in: )
+The command for pretraining DETReg on 8 GPUs on ImageNet100 is as following:
+- On MDef-DETR:
+```bash
+GPUS_PER_NODE=8 ./tools/run_dist_launch.sh 8 ./configs/DETReg_top30_in100.sh --pseudo_labels mdef_detr --batch_size 24 --num_workers 8
+```
+- On MDETR:
+```bash
+GPUS_PER_NODE=8 ./tools/run_dist_launch.sh 8 ./configs/DETReg_top30_in100.sh --pseudo_labels mdetr --batch_size 24 --num_workers 8
```

Comment:
Let's have a different training config (`.sh`) file for MDETR & MDef-DETR where we can specify different output directory names corresponding to the model.